src/QMCHamiltonians/ECPComponentBuilder.h

```diff
@@ -76,6 +80,11 @@ struct ECPComponentBuilder : public MPIObjectBase, public QMCTraits
                  const Matrix<mRealType>& vnn,
                  RealType rmax,
                  mRealType Vprefactor = 1.0);
+ 
+  void buildSO(const std::vector<int>& angList,
```

Comment:
documentation

Reply:
Added documentation.

Reply:
Please write doxygen style and document input parameters.